src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import { Routes, Route, Link } from "react-router-dom";
+import { auth } from "../firebase";
+import { onAuthStateChanged } from "firebase/auth";
+import logo from "../logo.png";
+import "./App.css";
+import Composer from "./Composer.js";
+import NewsFeed from "./NewsFeed.js";
+import AuthForm from "./AuthForm.js";
+
+const App = () => {
+  const [loggedInUser, setLoggedInUser] = useState();
```

Comment:
Since you set the loggedInUser state to null in line 18, might as well define a default of null here.
```suggestion
  const [loggedInUser, setLoggedInUser] = useState(null);
```